multi_explore_swarm_simulation_tb0:
  ros__parameters:
    robot_name: "tb0" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb1:
  ros__parameters:
    robot_name: "tb1" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb2:
  ros__parameters:
    robot_name: "tb2" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb3:
  ros__parameters:
    robot_name: "tb3" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb4:
  ros__parameters:
    robot_name: "tb4" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb5:
  ros__parameters:
    robot_name: "tb5" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb6:
  ros__parameters:
    robot_name: "tb6" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb7:
  ros__parameters:
    robot_name: "tb7" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb8:
  ros__parameters:
    robot_name: "tb8" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0

multi_explore_swarm_simulation_tb9:
  ros__parameters:
    robot_name: "tb9" #not used if simulation_mode == True, only used for real robot case, stores the unique robot name for each robot machine
    # peer_list: ["tb0", "tb1"]
    peer_list: ["tb0", "tb1","tb2", "tb3","tb4", "tb5","tb6", "tb7","tb8", "tb9"]

    tb0_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb1_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb2_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb3_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb4_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb5_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb6_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb7_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb8_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    tb9_init_offset: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

    simulation_mode: True  
    pid: 0